tests/unit/src/test/scala/tests/CodeLensesSlowSuite.scala

```diff
@@ -0,0 +1,41 @@
+package tests
+import scala.concurrent.Future
+
+object CodeLensesSlowSuite extends BaseSlowSuite("codeLenses") {
+  testAsync("run") {
+    for {
+      _ <- server.initialize(
+        """|/metals.json
+           |{
+           |  "a": { }
+           |}
+           |
+           |/a/src/main/scala/Main.scala
+           |object Main {
+           |  def main(args: Array[String]): Unit = {}
+           |}""".stripMargin
+      )
+      _ <- verifyCodeLenses(
```

Comment:
by convention, we use "assert" as a prefix for functions that compare two strings: `assertCodeLens`. This would be consistent with for example `assertCompletion`